src/nuget.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <clear />
+    <add key="VideoStreamingComponents" value="https://susohoni.pkgs.visualstudio.com/VideoStreamer/_packaging/VideoStreamingComponents/nuget/v3/index.json" />
+	<add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+  </packageSources>
+  <packageSourceCredentials>
+    <VideoStreamingComponents>
+      <add key="Username" value="susohoni@microsoft.com" />
+      <add key="Password" value="AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAA6GW3drdzR0ySsdslEU9aZwAAAAACAAAAAAAQZgAAAAEAACAAAADvObbtSvcanGh305J7dwv8vih6yZNGRE6tatP3W1JjAQAAAAAOgAAAAAIAACAAAADKNfn02ulD6y25CvFndhMOFV8PV8gM3VeY1AlHw0mrt0AAAADee5kwUZxCQPjtwD1fFSA5JsIXVfGeZnW8Vx45Uul+GS6WGDyCpyO4M5dl7quaZytUCnRSRku5NSxNYtsbNNUAQAAAALblK9qzQQ0DgEAQpJzb+YNEVihU1ql/l94pvGTSIlka3qZJt50yz1ODmgVt1+4BqsUDvD2kkhKcC8JUOHJ4nQw=" />

```

Comment:
><add key="Password" value="AQAAANCMnd8BFdERjHoAwE/Cl+sBAAAA6GW3drdzR0ySsdslEU9aZwAAAAACAAAAAAAQZgAAAAEAACAAAADvObbtSvcanGh305J7dwv8vih6yZNGRE6tatP3W1JjAQAAAAAOgAAAAAIAACAAAADKNfn02ulD6y25CvFndhMOFV8PV8gM3VeY1AlHw0mrt0AAAADee5kwUZxCQPjtwD1fFSA5JsIXVfGeZnW8Vx45Uul+GS6WGDyCpyO4M5dl7quaZytUCnRSRku5NSxNYtsbNNUAQAAAALblK9qzQQ0DgEAQpJzb+YNEVihU1ql/l94pvGTSIlka3qZJt50yz1ODmgVt1+4BqsUDvD2kkhKcC8JUOHJ4nQw=" /> [](start = 6, length = 403)

is this publicly disclosed??

Reply:
This token is disabled. Just a placeholder when actually using privately. The fork is private and this PR is not going to merge with this file.